pkg/cmd/convert/convert_remoteTasks.go

```diff
@@ -0,0 +1,138 @@
+package convert
+
+import (
+	"github.com/jenkins-x-plugins/jx-pipeline/pkg/lighthouses"
+	"github.com/jenkins-x-plugins/jx-pipeline/pkg/pipelines/processor"
+	"github.com/jenkins-x/jx-helpers/v3/pkg/cmdrunner"
+	"github.com/jenkins-x/jx-helpers/v3/pkg/cobras/helper"
+	"github.com/jenkins-x/jx-helpers/v3/pkg/cobras/templates"
+	"github.com/jenkins-x/jx-helpers/v3/pkg/gitclient"
+	"github.com/jenkins-x/jx-helpers/v3/pkg/gitclient/cli"
+	"github.com/jenkins-x/jx-helpers/v3/pkg/options"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"os"
+	"path/filepath"
+)
+
+// RemoteTasksOptions contains the command line options
+type RemoteTasksOptions struct {
+	options.BaseOptions
+	lighthouses.ResolverOptions
+
+	Dir           string
+	Namespace     string
+	TasksFolder   string
+	Format        string
+	UseSHA        string
+	CatalogSHA    string
+	Catalog       bool
+	UseKptRef     bool
+	TriggerCount  int
+	Processor     processor.Interface
+	CommandRunner cmdrunner.CommandRunner
+	GitClient     gitclient.Interface
+
+	// KptPath is the imported path in the catalog for repository pipelines
+	KptPath string
+}
+
+var (
+	remoteTasksCmdLong = templates.LongDesc(`
+		Converts the pipelines to use the 'image: uses:sourceURI' include mechanism
+
+	So that pipelines are smaller, simpler and easier to upgrade pipelines with the version stream
+`)
+
+	remoteTasksCmdExample = templates.Examples(`
+		# Convert a repository created using the alpha/beta of v3 
+        # to use the nice new uses: syntax 
+		jx pipeline convert
+
+		# Convert a pipeline catalog to the uses syntax and layout
+		jx pipeline convert --catalog
+	`)
+)
+
+// NewCmdPipelineConvertRemoteTasks creates the command
+func NewCmdPipelineConvertRemoteTasks() (*cobra.Command, *RemoteTasksOptions) {
+	o := &RemoteTasksOptions{}
+
+	cmd := &cobra.Command{
+		Use:     "remoteTasks",
```

Comment:
Don't use upper case in commands. It would be confusing since no other jx command (to my knowledge use) anything else than lower case.